stl/inc/xutility

```diff
@@ -4029,6 +4029,10 @@ template <class _InIt, class _OutIt>
 _CONSTEXPR20 _OutIt _Copy_unchecked(_InIt _First, _InIt _Last, _OutIt _Dest) {
     // copy [_First, _Last) to [_Dest, ...)
     // note: _Copy_unchecked has callers other than the copy family
+    if constexpr (_Is_vb_iterator<_InIt> && _Is_vb_iterator<_OutIt>) {
+        return _Copy_vbool(_First, _Last, _Dest);
+    }

```

Comment:
I'll report the "unreachable code" warning as a compiler bug, since the C4702 suppression in the STL isn't being properly respected in the header unit scenario. However, you can and should perma-workaround it here. As this is an `if constexpr return`, the following code is completely dead when the `return` is taken. For runtime `if`, we prefer early `return` without `else` increasing the following indentation, but for `if constexpr` we should use `else` to avoid compiling dead code that won't be taken. I have verified that this cures the header unit test warnings.

Reply:
Actually, I think this is a duplicate of VSO-1464637 "Standard Library Header Units: `#pragma warning` doesn't always suppress warnings in templates" whose fix will be shipping soon in 17.2 Preview 2.

Reply:
I have simply added the perma workaround without any more transition comments, as it is not really needed to "clean" anything up there